Sources/Components/Expandable/Expandable.swift

```diff
@@ -0,0 +1,201 @@
+import SwiftUI
+
+extension Warp {
+    /**
+
+     Expandable allows users to interact with expandable/collapsible sections of content. Also known as “accordion”
+
+     **When to use**
+
+     Use the Expandable component when presenting content that can be optionally revealed to users.
+     Consider it for long content that may overwhelm users if displayed all at once.
+
+     **Usage**
+
+     Provide the `Warp.Expandable` with:
+
+     - A `Warp.ExpandableStyle`, choose from:
+     - `default`: for a non bordered expandable.
+     - `box`: for a bordered expandable with rounded corners.
+     - `boxBleed`: for a bordered expandable with sharp corners.
+     - A `title`.
+     - Either:
+       - A `subtitle` String
+       - An `expandableView` `View`
+     - A boolean `isAnimated` to determine if the expand/collapse transition is animated or not. **Optional** _default value is `true` if none is specified_.
+     - A `ColorProvider`. **Optional:** _default is read from `Config.colorProvider` if none is specified_.
+     */
+    public struct Expandable<Content: View>: View {
+        /**
+         **Collapsed by default**
+
+         Expendables begins in the collapsed state with all content panels closed.
```

Comment:
It seems on web component can start from expanded state also, so maybe we should follow suit and add that as a param.

<img width="737" alt="Screenshot 2024-07-03 at 10 21 25" src="https://github.com/warp-ds/warp-ios/assets/89784012/82470e65-117f-4588-bf81-894f8396e0f7">

Reply:
Ha! I had initially exposed the parameter so you could start expanded 😅 

No problem, I'll do so again 👍